bin/git-create-branch

```diff
@@ -18,7 +26,7 @@ do
 	shift
 done
 
-if [[ -z $BRANCH ]] && [[ -n $REMOTE ]] 
+if [[ -z $BRANCH ]] && [[ -n $REMOTE ]]
```

Comment:
I just found an edge case. If the user runs `git create-branch` by mistake. he/she will get the preference remote created as the branch name.

Reply:
great catch. now fixed